src/utils/user-agent.ts

```diff
@@ -1,6 +1,9 @@
 export const userAgentParser = (rawRegex: string, userAgent: string): string[] | null => {
-  const regex = `(?:^|[^A-Z0-9-_]|[^A-Z0-9-]_|sprd-)(?:${rawRegex})`;
-  const match = RegExp(regex, "i").exec(userAgent);
-
-  return match ? match.slice(1) : null;
+  try {
+    const regex = `(?:^|[^A-Z0-9-_]|[^A-Z0-9-]_|sprd-)(?:${rawRegex})`;
+    const match = RegExp(regex, "i").exec(userAgent);
+    return match ? match.slice(1) : null;
```

Comment:
Can you run `npm run lint` and make sure everything's green? Tslint should complain about the return statement not having a new line before it.